lib/cli/commands/build.js

```diff
@@ -52,28 +53,31 @@ optimized output compatible with Node 6.
     });
   }
 
-  build(dirs = this.dirs) {
-    if (!dirs) {
-      dirs = this.dirs = [ 'app', 'config', 'blueprints' ];
+  build(globs = this.globs) {
+    if (!globs) {
+      globs = this.globs = [ 'app/**/*', 'config/**/*', 'blueprints/*/index.js' ];
```

Comment:
Not sure we have enough use-cases to justify making the change now (otherwise we'd be rewriting it twice if we don't plan accordingly), but yes I do think it has to be done eventually.